app/service/sorting_test.go

```diff
@@ -137,13 +137,13 @@ func TestApplySorting(t *testing.T) {
 			shouldPanic: errors.New(`unsupported type "interface{}" for field "name"`)},
 		{name: "unallowed from fields",
 			args: args{
-				urlParameters: "?from.field=Joe&from.version=2",
+				urlParameters: "?from.field=Joe&from.version=2&from.name=Jane",
```

Comment:
As there is this "version", it may test something else. Probably it should be 2 different tests (?)

Reply:
Both 'version' and 'name' are not listed in `sort`, so here they should be similar (`acceptedFields` should not matter). That's why they are both in the same test.